lib/travis/build/templates/header.sh

```diff
@@ -12,6 +12,13 @@ ANSI_GREEN="\033[32;1m"
 ANSI_RESET="\033[0m"
 ANSI_CLEAR="\033[0K"
 
+: "${SHELL:=/bin/bash}"
+: "${TERM:=xterm-256color}"
```

Comment:
I don't think it supports 256.  I think I misinterpreted something said in chat.  Checking...